packages/react-core/codeConnect/icons/IconWrapper.figma.tsx

```diff
@@ -0,0 +1,58 @@
+import { IconWrapper } from 'src/components/Icon/IconWrapper.tsx';
+import figma from '@figma/code-connect';
+
+figma.connect(
+  IconWrapper,
+  'https://www.figma.com/design/VMEX8Xg2nzhBX8rfBx53jp/branch/9QzcMKMR7FX7HXlc0sRUSp/PatternFly-6--Components?node-id=198-1573&t=dP3Et2mI3FtI6IEP-11',
+  {
+    props: {
+      // Icon instance from Figma
+      icon: figma.instance('🖼️ Icon'),
+
+      // Comprehensive icon size mapping
+      iconSize: figma.enum('Icon Size', {
```

Comment:
We can default to the "size" prop rather than "iconSize" prop for consistency with docs examples.
```suggestion
      size: figma.enum('Icon Size', {
```